src/app/page/component/page/tract-page/tract-page.component.html

```diff
@@ -8,31 +8,33 @@
       hasPageHeader: hasPageHeader
     }"
   >
-    <app-page-new-header
+    <app-page-header
       *ngIf="header && (isModal$ | async) === false"
       [header]="header"
-    ></app-page-new-header>
-    <app-page-new-hero
+    ></app-page-header>
+    <app-page-hero
       *ngIf="hero && (isModal$ | async) === false"
       [hero]="hero"
-    ></app-page-new-hero>
+    ></app-page-hero>
+    <app-page-hero
+      *ngIf="content && (isModal$ | async) === false"
+      [hero]="{ content: content }"
+    >
```

Comment:
Do you think we could simplify this or not? It's okay if we can't it just looked like duplicated code.
```suggestion
    <app-page-hero
      *ngIf="(isModal$ | async) === false && (hero || content)"
      [hero]="hero || { content: content }"
    ></app-page-hero>
    >
```

Reply:
I will revert the commit that adds content to the hero because that was from when we were going to use the tract-page for CYOA tools.